examples/tutorial_derive/04_01_enum.rs

```diff
@@ -10,6 +10,7 @@ struct Cli {
 
 #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, ArgEnum)]
 enum Mode {
+    #[clap(name="f")]
```

Comment:
1. The part of the tutorial dealing with this just introrduced the concept.  After the initial example it can make sense to call out attributes exist to customize it, with a relative cross-link to the derive reference.  I'm unsure about actually including an example though.
2. This tutorial is a parallel of the builder one and they need to be kept in sync